### PR TITLE
Ut2 integration

### PR DESCRIPTION
**Pull request by spangenberg**

### CODE REVIEW

**Comment by squat** [comment]
how is the example generated?

File: installer/pkg/config/types.go [hunk 1]
```diff
@@ -10,63 +10,85 @@ import (
...
-
 type containerLinux struct {
 	Channel string `yaml:"Channel"`
 	Version string `yaml:"Version"`
 }
 
-type dns struct {
-	BaseDomain string `yaml:"BaseDomain"`
+type ddns struct {
+	Key    ddnsKey `yaml:"Secret"`
```
Comment (squat) [inline installer/pkg/config/types.go]: both these fields have the same key

File: installer/pkg/config/types.go [hunk 2]
```diff
@@ -10,63 +10,85 @@ import (
...
-	NodeCount       int      `yaml:"NodeCount"`
-	MachineType     string   `yaml:"MachineType"`
-	ExternalServers []string `yaml:"ExternalServers"`
+	Count    int          `yaml:"Count"`
+	External etcdExternal `yaml:"External"`
+}
+
+type etcdExternal struct {
+	CACertPath     string   `yaml:"CACertPath"`
+	ClientCertPath string   `yam:"ClientCertPath"`
```
Comment (squat) [inline installer/pkg/config/types.go]: this struct tag should say `yaml`

**Comment by mxinden** [comment]
@squat As far as I understood, this is manually written. We should invest into generating these soon. Maintaining them manually would be impossible in the long run.

**Comment by mxinden** [comment]
@spangenberg Kubernetes uses camelCasing, not PascalCasing for any yaml file. Any thoughts on this?

File: installer/pkg/config/types.go (hunk 2 above)
Comment (spangenberg) [reply installer/pkg/config/types.go]: will fix

**Comment by spangenberg** [comment]
@mxinden we already spoke about renaming everything. As soon as the structure is fixed I'm going to rename everything.

**Comment by squat** [comment]
@mxinden I raised the same issue with casing last week. It is very tedious to write all the caps and it is also inconsistent with all other yaml we see: k8s, container linux config etc

**Comment by mxinden** [comment]
@squat are you ok with proceeding here and following up with the casing before merging `ut2-integration` into master?

**Comment by mxinden** [comment]
@squat Thanks.

**Pull request merged**